serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,37 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
+
+In {ServerlessProductName} 1.4.0 and newer versions, the `KnativeServing` object's API group changed from `serving.knative.dev` to `operator.knative.dev`.
+You must adjust any of your scripts or applications that rely on the old API group to use the new API group.
+
+Once you have updated references to the new API group, you must remove any older custom resource (CR) versions and use the newly deployed `KnativeServing` CR instead.
+
+To safely do this without downtime, remove the owner reference from the newly deployed `KnativeServing` CR by entering the following command:
+----
+ $ oc edit knativeserving.operator.knative.dev knative-serving -n knative-serving
+----
+
+After the owner reference has been removed, you can safely remove any older CR versions and start using the new one.
+
+[IMPORTANT]
+====
+If a previous version of the CR exists, changes to the new CR will be overwritten by the {ServerlessOperatorName}. While the old CR is still active, all changes need to be made to that CR.
+====
+
+This change to the API group causes commands without a fully qualified API group and kind, such as `oc get knativeserving`, to become unreliable and not always work correctly.
```

Comment:
~I think this might make more sense after line 14.~ If this is only temporary, I might present it slightly differently. It goes from a "you need to know to not do this" fact to just extra context information that you could present after the step to remove the old CRD.

Reply:
I moved the note as requested, but I'm not sure if there's something else you are asking me to do with this?